src/utils/data2json.sh

```diff
@@ -10,6 +10,7 @@ lang=""
 feat="" # feat.scp
 oov="<unk>"
 bpecode=""
+verbose=1
```

Comment:
You can use `verbose=0` as default, and remove `--verbose ${verbose}` in `run.sh`